### ISSUE TITLE
 Pretrained ViFi-CLIP Model Weights

### ISSUE THREAD

**Issue by SHIBOYA** [muzairkhattak/ViFi-CLIP#13]
Hello,

I am interested in your work on ViFi-CLIP as presented in your GitHub repository. I am particularly looking to download the pretrained model weights for the ViFi-CLIP model, specifically the weights trained on K-400 used in VL prompting approach mentioned in the base-to-novel setting instructions. That is “TRAINED-K400/VIFI-CLIP-CKPT” in the following command
"python -m torch.distributed.launch --nproc_per_node=8 main.py \
-cfg configs/base2novel/prompting_base2novel/hmdb/16_32_prompting_s1.yaml --output /PATH/TO/OUTPUT \ 
--resume /PATH/TO/TRAINED-K400/VIFI-CLIP-CKPT"

I checked the repository but was unable to find a direct link to these pretrained model weights. Could you kindly guide me on how I can obtain these weights? Are they available for public download?

Thank you for your assistance and for the impressive work on this project.

Best regards！

**Comment by muzairkhattak** [comment]
Hi @SHIBOYA,

Thank you for showing interest in ViFi-CLIP,

The links to the pre-trained weights are given in the main README.file of the repository. 

Specifically, the download links are given in the respective result's table  [at this link.](https://github.com/muzairkhattak/ViFi-CLIP#base-to-novel-generalization-results)

Please let us know if you face any difficulties in downloading the pre-trained weights.

Thank you and kind regards,
Muhammad Uzair

**Comment by SHIBOYA** [comment]
I wanted to extend my gratitude for your prompt and helpful response regarding the pretrained ViFi-CLIP model weights. 

Thank you once again for your support and for the remarkable work on the ViFi-CLIP project.

**Comment by SHIBOYA** [comment]
I have a follow-up question regarding the evaluation metrics used in the Base-to-Novel Generalization Setting of your paper.

Could you please clarify whether the evaluation metric used for this setting is top-1 accuracy, top-5 accuracy, or another metric? This information would be vital for a comprehensive understanding of your results.

Additionally, would it be possible for you to release the training logs for this particular experiment? Access to these logs would be incredibly helpful for replicating your results and understanding the model's performance over the training period.

Thank you once again for your time and assistance.

Best regards!
